js/private/translate_package_lock.bzl

```diff
@@ -105,7 +116,7 @@ _NPM_IMPORT_TMPL = """\
         integrity = "{integrity}",
         package = "{package}",
         version = "{version}",
-        deps = {deps},
+        deps = {deps},{maybe_patches}{maybe_patch_args}
```

Comment:
A bit awkward, but I didn't want to output `patches = [...]` and `patch_args = [...]` if there weren't any.

Reply:
yeah I was commenting about it but came to the same conclusion, sometimes the generated thing will be read